packages/headless/src/utils/query-expression/exact-match/exact-match.ts

```diff
@@ -2,6 +2,9 @@ import {getNegationPrefix, Negatable} from '../common/negatable';
 import {Part} from '../common/part';
 
 export interface ExactMatchExpression extends Negatable {
+  /**
+   * An expression that must appear in its entirety at least once for an item to be returned.
+   */
```

Comment:
I think we should give examples in this and `KeywordExpression` to help differentiate them.